po/plume/de.po

```diff
@@ -315,12 +315,12 @@ msgstr ""
 msgid "Articles"
 msgstr "Artikel"
 
-msgid "Followers"
-msgstr "Follower"
+msgid "Subscribers"
+msgstr ""
 
 #, fuzzy
-msgid "Followed"
-msgstr "Folgen"
+msgid "Subscriptions"
+msgstr "Beschreibung"
```

Comment:
not quite… "Abonoment" would be a better translation.

Reply:
Yes, these strings have probably been automatically translated by GetText… :confused: I will edit them.